### DISCUSSION TITLE
How to build an arm64 image?

### DISCUSSION THREAD

**Discussion by Serg046** [werf/werf#6313] [General]
I try to build [this image](https://github.com/Serg046/serg046.dev/blob/master/Dockerfile) with [this GH actions job](https://github.com/Serg046/serg046.dev/blob/master/.github/workflows/prod.yml). It works fine with buildx + docker/build-push-action@v6 but I cannot achieve the same with `werf converge`.

When I change [the current config](https://github.com/Serg046/serg046.dev/blob/master/werf.yaml)
```
project: serg046-app
configVersion: 1
```
to be
```
project: serg046-app
configVersion: 1
build:
  platform:
  - linux/arm64
---
image: app
dockerfile: Dockerfile
```
it starts failing with
```
Error: phase build on image app stage dockerfile handler failed: failed to build image for stage dockerfile with digest cc74bb3a75ea170ee88e82877fe0dddd82e392088bd3118accc81c77: error building dockerfile with docker-server-backend: failed to find driver "docker-container"
```
You can find the build [here](https://github.com/Serg046/serg046.dev/actions/runs/10393986323/job/28782728845). Is there a way to get rid of docker/build-push-action@v6 and ask werf to build instead?

**Comment by alexey-igrychev** [comment]
Hi. We encountered a similar issue (the same error) when running werf locally on Mac. In our case, the problem was resolved after updating Docker to the latest version.

**Comment by alexey-igrychev** [comment]
It seems it should work fine if you use https://github.com/docker/setup-buildx-action before running werf.

**Comment by alexey-igrychev** [comment]
Fixed in this PR https://github.com/werf/werf/pull/6760

**Comment by Serg046** [comment]
Thanks guys, works as expected
